src/interfaces/uproxy_core_api.ts

```diff
@@ -30,6 +30,12 @@ export enum UserFeedbackType {
   OTHER_FEEDBACK = 9
 }
 
+// Object containing an update to apply to another object
```

Comment:
This is extremely abstract. Many other types here make it explicit for which operation they are, e.g. `CloudOperationArgs`. How about `UpdateGlobalSettingArgs`?